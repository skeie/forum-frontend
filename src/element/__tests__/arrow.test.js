import React from 'react';
import Arrow from '../arrowUI';
import posts from '../../testData/posts';
import renderer from 'react-test-renderer';

const fetch = jest.fn();

it('Arrow renders correctly left way', () => {
    const tree = renderer.create(<Arrow way="left" fetch={fetch} />).toJSON();
    expect(tree).toMatchSnapshot();
});

it('Arrow renders correctly right way', () => {
    const tree = renderer.create(<Arrow way="right" fetch={fetch} />).toJSON();
    expect(tree).toMatchSnapshot();
});
