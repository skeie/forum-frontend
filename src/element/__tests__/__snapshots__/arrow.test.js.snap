// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Arrow renders correctly left way 1`] = `
<ActivityIndicator
  animating={true}
  color="white"
  hidesWhenStopped={true}
  size="small"
/>
`;

exports[`Arrow renders correctly right way 1`] = `
<ActivityIndicator
  animating={true}
  color="white"
  hidesWhenStopped={true}
  size="small"
/>
`;
